Package.swift

```diff
@@ -1,4 +1,4 @@
-// swift-tools-version:5.3
+// swift-tools-version:5.9
```

Comment:
Because I do use the new access modifier `package` https://forums.swift.org/t/new-access-modifier-package/61459 I did need to bump the min swift version; If you do not want to upgrade I can change the code marked with `package` to `public`.